@navikt/core/css/darkside/form/combobox.darkside.css

```diff
@@ -117,12 +118,12 @@
 
 .navds-combobox--error {
   & .navds-combobox__wrapper-inner {
-    border-color: var(--ax-border-danger-strong);
-    border-width: 2px;
-    outline-offset: 2px;
+    border-color: var(--ax-border-danger);
+    box-shadow: 0 0 0 1px var(--ax-border-danger);
+    outline-offset: 3px;
```

Comment:
Don't think this does anything atm

Reply:
Tror den gjorde at "border" ble ekstra tykk? 🤔 
med box-shadow:
![image](https://github.com/user-attachments/assets/8d14fc64-1a5a-4fe0-916f-e0fbb5fdb052)
uten box-shadow:
![image](https://github.com/user-attachments/assets/a062b1ca-2edd-4886-84b5-3b8942228ebf)

Reply:
oh 🤦 , du mente outline-offset

Reply:
ja den kan fjernes 🤔, skal jeg dytte en slik på main? (blir litt mye med en hel PR bare for det, kan pair progge den for safety)